packages/fx-core/src/common/constants.ts

```diff
@@ -35,3 +35,25 @@ export class FeatureFlagName {
   static readonly BicepEnvCheckerEnable = "TEAMSFX_BICEP_ENV_CHECKER_ENABLE";
   static readonly APIV2 = "TEAMSFX_APIV2";
 }
+
+export class ArmParameters {
```

Comment:
If these constants are only used in migration, can we put them to the migration middleware's file?

Reply:
sure